server/src/main/java/io/crate/metadata/upgrade/IndexTemplateUpgrader.java

```diff
@@ -170,4 +178,79 @@ private static void populateColumnPositions(String parentName,
                                     maxColumnPosition);
         }
     }
+
+    // bug : https://github.com/crate/crate/issues/13380
+    private static Map<String, IndexTemplateMetadata> fixCorruptedTemplatesFromSwapTableBug(Map<String, IndexTemplateMetadata> templates) {
```

Comment:
when template names start with `.partitioned`, either the name is corrected or find the partner template to complete the swap that was affected by the bug.